pkg/security/podsecuritypolicy/provider.go

```diff
@@ -296,6 +296,28 @@ func (s *simpleProvider) ValidatePod(pod *api.Pod) field.ErrorList {
 							"Flexvolume driver is not allowed to be used"))
 				}
 			}
+
```

Comment:
This function is getting hard to read - mind pulling the volume loop body out to a separate function?